TurkishFilter/sections/specific.txt

```diff
@@ -64,6 +64,10 @@ seskocaeli.com,kocaelizirve.com,yenialanya.com,t54.com.tr,haber266.com,safihaber
 ||yeniasya.com.tr/sites/yeniasya/static/*-kampanya
 !
 belge.com.tr##.haberdetayReklam
+||elitbursa.com/wp-content/uploads/*/besas-*.jpg
+||elitbursa.com/wp-content/uploads/*/banner.jpg
+elitbursa.com##.widget_media_image
+elitbursa.com##.widget_text
```

Comment:
wide rules. Block ads by URL(one regexp, for example)

Reply:
Leftovers will remain in this case, perhaps it's better to use ext-css?
(images are small)

Reply:
this site has few visitors